desktop/core/src/desktop/js/api/apiHelper.js

```diff
@@ -191,20 +190,16 @@ class ApiHelper {
    * @param {string} id
    * @param {*} [value] - Optional, undefined and null will remove the value
    */
-  setInTotalStorage(owner, id, value) {
+  setInLocalStorage(owner, id, value) {
```

Comment:
Nit: Even this function looks like a better fit in a storageUtils.ts than apiHelper.js

Reply:
Yes. ApiHelper is a monster to tackle gradually, it stems from the time when we didn't have modularization on the js side. The idea I have here is to move the API-related functionality closer to each app, notebook, table browser etc. and get rid of the localStorage logic here, the extra "owner" and "id" aren't providing any value so better to drop them.